Packs/Logzio/Integrations/Logzio/Logzio.py

```diff
@@ -0,0 +1,256 @@
+                time_filter["include_upper"] = True
+            payload["query"]["bool"]["must"].append(
+                {
+                    "range": {"@timestamp": time_filter}
+                }
+            )
+        response = self.execute_api(SEARCH_LOGS_API_SUFFIX, payload, self.op_api_token)
+        return response.get("hits", {}).get("hits", {})
+
+    def get_rule_logs(self, id, size, page_size=MAX_LOGZIO_DOCS):
+        payload = {
+            "filter": {
+                "alertEventId": id
+            },
+            "pagination": {
+                "pageNumber": 1,
+                "pageSize": page_size
+            }
+        }
+        response = self.execute_api(SEARCH_RULE_LOGS_API_SUFFIX, payload, self.security_api_token)
+        total = response.get("total", 0)
+        results = response.get("results", [])
+        if total > page_size and size > page_size:
+            for i in range(2, (min(size, total) + page_size - 1) // page_size + 1):  # Ceiling division
+                payload["pagination"]["pageNumber"] = i
+                response = self.execute_api(SEARCH_RULE_LOGS_API_SUFFIX, payload, self.security_api_token)
+                results += response.get("results", [])
+        return results
+
+    def get_base_api_url(self):
+        return BASE_URL.replace("api.", "api{}.".format(self.get_region_code()))
+
+    def get_region_code(self):
+        if self.region != "us" and self.region != "":
+            return "-{}".format(self.region)
+        return ""
+
+    def execute_api(self, url_suffix, payload, api_token):
+        headers = {
+            'Content-Type': 'application/json',
+            'X-API-TOKEN': api_token
+        }
+        return BaseClient._http_request(self, "POST", url_suffix, headers=headers, data=json.dumps(payload),
+                                        ok_codes=(200, ))
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+def test_module(client):
+    try:
+        client.fetch_triggered_rules()
+        client.search_logs('*', 10, "", "")
+        return 'ok'
+    except Exception as e:
+        return 'Test failed: {}'.format(e)
+
+
+def search_logs_command(client, args):
+    if client.op_api_token is None:
+        return_error("Operational API Token wasn't provided, cannot perform search")
+    query = args.get('query')
+    size = args.get('size', MAX_LOGZIO_DOCS)
+    from_time = args.get('from_time')
+    to_time = args.get('to_time')
+    resp = client.search_logs(query, size, from_time, to_time)
+    content = []
+    for res in resp:
+        log = res.get("_source", None)
+        log_context = {
+            'type': log.get("type", None),
+            'timestamp': log.get("@timestamp", None),
+            'content': log
+        }
+        content.append(log_context)
+    if len(content) == 0:
+        context = None
+        readable = '### No logs were found'
+    else:
+        context = {
+            'Logzio.Logs': content
```

Comment:
Please change this (also in the yml) to `Logzio.Result` to be consistent to other SIEM integrations (i.e. Splunk)